frame/babel/Cargo.toml

```
@@ -41,11 +42,13 @@ pallet-evm-precompile-blake2 = { workspace = true, optional = true }
 pallet-evm-precompile-bn128 = { workspace = true, optional = true }
 pallet-evm-precompile-modexp = { workspace = true, optional = true }
 pallet-evm-precompile-simple = { workspace = true, optional = true }
+pallet-solana = { workspace = true }
```

```suggestion
pallet-solana = { workspace = true, optional = true }
```
- Add `pallet-solana` to the feature "solana"
- Add `pallet-solana = { workspace = true, default-features = true}` to `[dev-dependencies]`.